internal/pkg/cli/app_init.go

```diff
@@ -232,6 +234,14 @@ func (o *initAppOpts) validateAppName(name string) error {
 	if err != nil {
 		var noSuchAppErr *config.ErrNoSuchApplication
 		if errors.As(err, &noSuchAppErr) {
+			roleName := fmt.Sprintf("%s-adminrole", name)
+			tags, err := o.iamRoleManager.ListRoleTags(roleName)
+			if err != nil {
+				return nil
+			}
+			if _, hasTag := tags[deploy.AppTagKey]; hasTag {
+				return fmt.Errorf("application named %s already exists in another region", name)
```

Comment:
Great move checking the Copilot tag!

Should we still error out even if they don't have the Copilot tag?  As long as the role exists (regardless whether it's created by Copi or not), Copilot won't be able to  create the application.